homeassistant/components/geizhals/manifest.json

```diff
@@ -2,6 +2,6 @@
   "domain": "geizhals",
   "name": "Geizhals",
   "documentation": "https://www.home-assistant.io/integrations/geizhals",
-  "requirements": ["geizhals==0.0.9"],
+  "requirements": ["geizhalscrawler==0.1"],
```

Comment:
This packages uses webscraping to gather information. This is not allowed and violated architectural decision record 0004 (ADR-0004).

See: <https://github.com/home-assistant/architecture/blob/master/adr/0004-webscraping.md>

Upstream code sample:

```py
        # parse prices
        self.device.prices = []
        for tmp in soup.select('div.offer__price .gh_price'):
            matches = re.search(_REGEX, tmp.string)
            raw = '{}.{}'.format(matches.group(1),
                                 matches.group(2))
            self.device.prices += [float(raw)]
```

<https://github.com/timherrm/geizhalscrawler/blob/60fe6b1db9344412541ce95235da5f8e61ff9a3a/geizhalscrawler/geizhalscrawler.py#L82>